cinema/serializers.py

```diff
@@ -1,24 +1,35 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import (
+    Movie,
+    Actor,
+    Genre,
+    CinemaHall,
+)
 
 
-class MovieSerializer(serializers.Serializer):
-    id = serializers.IntegerField(read_only=True)
-    title = serializers.CharField(max_length=255)
-    description = serializers.CharField()
-    duration = serializers.IntegerField()
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = "__all__"
 
-    def create(self, validated_data):
-        return Movie.objects.create(**validated_data)
 
-    def update(self, instance, validated_data):
-        instance.title = validated_data.get("title", instance.title)
-        instance.description = validated_data.get(
-            "description", instance.description
-        )
-        instance.duration = validated_data.get("duration", instance.duration)
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
 
-        instance.save()
 
-        return instance
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = "__all__"
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    actors = ActorSerializer(read_only=True, many=True)
+    genres = GenreSerializer(read_only=True, many=True)
```

Comment:
The task says: "Do not use related serializers for ManyToMany relations."
Think why it's there and what can go wrong (or see the relevant docs section ;)